embassy-nrf/src/pwm.rs

```diff
@@ -152,6 +423,12 @@ impl<'d, T: Instance> Pwm<'d, T> {
             .write(|w| unsafe { w.countertop().bits(duty.min(32767u16)) });
     }
 
+    /// Additional number of PWM periods spent on each duty cycle value.
```

Comment:
I'm not sure if this is useful for the "simple" PWM. Since the channels are always playing the same duty (that you set just once), playing them once or many times doesn't make a difference, right?